packages/@aws-cdk/aws-apigatewayv2/lib/api.ts

```diff
@@ -0,0 +1,264 @@
+import * as lambda from '@aws-cdk/aws-lambda';
+import * as cdk from '@aws-cdk/core';
+import * as apigatewayv2 from '../lib';
+import { CfnApiProps } from './apigatewayv2.generated';
+
+/**
+ * the HTTP API interface
+ */
+export interface IApi extends cdk.IResource {
```

Comment:
apigatewayv2 has HTTP and websocket APIs. The name `IApi` hints that it could refer to both but the description says that it isn't. Either this should become `IHttpApi` or this should be applicable to both HTTP APIs and web sockets.

Which is the right way?

Reply:
`IApi` is the interface for `CfnApi` and `IHttpApi` extends it for `HTTP APIs` with `HTTP Proxy Integration`.  I hope I've made it clear in the README.